crates/ruff_db/src/source.rs

```diff
@@ -16,7 +15,6 @@ pub fn source_text(db: &dyn Db, file: File) -> SourceText {
 
     SourceText {
         inner: Arc::from(content),
-        count: Count::new(),
```

Comment:
The count here was misleading because it incremented the count everytime `SourceText` was cloned